jac-cloud/scripts/Readme.md

```diff
@@ -0,0 +1,186 @@
+## Usage Documentation for jac-cloud Deployment
```

Comment:
Can you add this to rest of the jac documentation? @Jayanaka-98 and @savini98 can help point you towards how to do that.